<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>后台管理界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add_btn">
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_btn">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="one_empty">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>设置为空</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="all_empty">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>全空按钮</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>营业额</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="SUM"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>元</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="table_btn">
       <property name="text">
        <string>餐桌管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menu_btn">
       <property name="text">
        <string>菜单管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="order_btn">
       <property name="text">
        <string>订单管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="user_btn">
       <property name="text">
        <string>用户管理</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back">
       <property name="text">
        <string>返回登陆</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
